packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -1,249 +1,243 @@
+import { existsSync } from "fs";
+import { cp } from "fs/promises";
 import { join } from "path";
 import { retry } from "helpers/command";
+import { sleep } from "helpers/common";
+import { detectPackageManager } from "helpers/packages";
 import { fetch } from "undici";
-import { beforeAll, describe, expect, test } from "vitest";
+import {
+	afterEach,
+	beforeAll,
+	beforeEach,
+	describe,
+	expect,
+	test,
+} from "vitest";
 import { deleteProject, deleteWorker } from "../scripts/common";
 import { getFrameworkMap } from "../src/templates";
 import { frameworkToTest } from "./frameworkToTest";
 import {
+	createTestLogStream,
 	isQuarantineMode,
 	keys,
 	recreateLogFolder,
 	runC3,
+	spawnWithLogging,
 	testDeploymentCommitMessage,
 	testProjectDir,
+	waitForExit,
 } from "./helpers";
 import type { FrameworkMap, FrameworkName } from "../src/templates";
 import type { RunnerConfig } from "./helpers";
-import type { Suite, TestContext } from "vitest";
+import type { WriteStream } from "fs";
+import type { Suite } from "vitest";
 
 const TEST_TIMEOUT = 1000 * 60 * 5;
 const LONG_TIMEOUT = 1000 * 60 * 10;
 
-type FrameworkTestConfig = Omit<RunnerConfig, "ctx"> & {
-	expectResponseToContain: string;
+type FrameworkTestConfig = RunnerConfig & {
 	testCommitMessage: boolean;
-	timeout?: number;
 	unsupportedPms?: string[];
 	unsupportedOSs?: string[];
+	verifyDev?: {
+		route: string;
+		expectedToken: string;
```

Comment:
Nit (and a very personal opinion 😅): I feel like `expectedToken` is a bit misleading and that something like `expectedText` or something like that could be more clear